sources/packages/api/src-sql/ProgramYear/Create-program-year.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS program_year(
```

Comment:
In general, the table names are plural. Can you please change the table name to "program_years" and change the rollback script as well.